README.md

```diff
@@ -84,7 +84,7 @@ A collection of powerful books that can help at different points in life
 
 - https://css-tricks.com/ - A front-end blog with up to date topics that release weekly
 - IDS Dropdown typeahead - When users need to select from a long list of options, use the drop-down typeahead. As soon as the user starts typing, the list changes to show suggestions that should get closer to what the user wants.
-- https://stateofjs.com/ - State of JS 20XX
+- https://stateofjs.com/ - State of JS 2018
```

Comment:
🤷‍♀️  I did not make this change...? I'll try to revert it manually and see what happens